.vscode/settings.json

```diff
@@ -16,8 +16,13 @@
     ".releaserc": "json"
   },
   "omnisharp.autoStart": false,
-  "jest.autoRun": "off",
-  "jest.jestCommandLine": "pnpm jest",
+  "jest.runMode": "on-demand",
+  "jest.jestCommandLine": "node --experimental-vm-modules node_modules/jest/bin/jest.js",
```

Comment:
doesn't this cause import issues with pnpm?

Reply:
Just managed to run the entire test set from the UI with no problems 🤷‍♂️